MAUI/DataGrid/grid-events.md

```diff
@@ -38,6 +38,24 @@ private void dataGrid_CellTapped(object sender, DataGridCellTappedEventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+### CellTappedCommand and its Parameter
+The command will be invoked while tapping a cell in the SfDataGrid. The instance of `DataGridCellTappedEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellTappedCommandParameter` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid CellTappedCommand="{Binding ButtonCommand}"
+                       ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public Command ButtonCommand { get; } = new Command(ListenTap);
+
+private static void ListenTap(object obj)
+{
+    System.Diagnostics.Debug.WriteLine(obj.GetType().Name);
+}
```

Comment:
Command TapCommand = new Command(ListenTap);

private static void ListenTap(object obj)
{
    var args =  obj as DataGridCellTappedEventArgs;
    if (args != null) 
    {
    var rowIndex = e.RowColumnIndex.RowIndex;
    var rowData = e.RowData;	
    var columnIndex = e.RowColumnIndex.ColumnIndex;
    var column = e.Column;
    }
}

Reply:
Changed like this